### DISCUSSION TITLE
Newby question: where to download the Macos-DMG installer for the 4.0.6 release

### DISCUSSION THREAD

**Discussion by dustinspace** [utmapp/UTM#4490] [General]
Hi everybody,

This is a really newly question ...
is there a repository with pre/build images to download? Or the only way is building the pre-release from the source code?

Thanks a lot.

**Comment by easp** [comment]
Look under "Assets" for each release. 
https://github.com/utmapp/UTM/releases

**Comment by dustinspace** [comment]
WHOW ! :-D it was easy but I never scrolled till the end of the page.
Thanks @easp :-D

**Comment by koshkin-ccna** [comment]
❯ brew info utm-beta

> ==> utm-beta: 4.0.7
> https://mac.getutm.app/
> Not installed
> From: https://github.com/Homebrew/homebrew-cask-versions/blob/HEAD/Casks/utm-beta.rb
> ==> Name
> UTM
> ==> Description
> Virtual machines UI using QEMU
> ==> Artifacts
> UTM.app (App)
